webserver/templates/data/summary.html

```diff
@@ -0,0 +1,107 @@
+{%- extends 'base.html' -%}
+
+{%- block title -%}
+  Track "{{ metadata['title'] }}" by {{ metadata['artist'] }} - AcousticBrainz
+{%- endblock -%}
+
+{%- block content -%}
+  <div id="page-recording-summary">
+
+    <h2 class="page-title">Track "{{ metadata['title'] }}" by {{ metadata['artist'] }}</h2>
+
+    {%- include 'data/summary-metadata.html' -%}
+
+    {% if submission_count > 1 %}
+      <h3 class="text-center">
+        {% if offset > 0 %}
+          <a href="{{ url_for('data.summary', mbid=metadata['mbid'], n=offset-1) }}"
+             title="View previous submission">&larr;</a>
+        {% endif %}
+        Submission #{{ offset + 1 }} <small>out of {{ submission_count }}</small>
+        {% if offset < submission_count - 1 %}
+          <a href="{{ url_for('data.summary', mbid=metadata['mbid'], n=offset+1) }}"
+             title="View next submission">&rarr;</a>
+        {% endif %}
+      </h3>
+    {% endif %}
+
+    <h3>Low-level information <small>Summary</small></h3>
+
+    <table class="table table-striped table-bordered">
+      <thead><tr><th style="width: 30%;">Tonal & Rhythm</th><th>value</th></tr></thead>
+      <tr>
+        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_ChordsDescriptors.html">key</a></td>
+        <td>{{ data.lowlevel.tonal.key_key }}</td>
+      </tr>
+      <tr>
+        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_ChordsDescriptors.html">scale</a></td>
+        <td>{{ data.lowlevel.tonal.chords_scale }}</td>
+      </tr>
+      <tr>
+        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_Danceability.html">danceability</a></td>
+        <td>{{ data.lowlevel.rhythm.danceability }}</td>
+      </tr>
+      <tr>
+        <td><a href="http://essentia.upf.edu/documentation/reference/streaming_RhythmExtractor2013.html">bpm</a></td>
+        <td>{{ data.lowlevel.rhythm.bpm }}</td></tr>
+      <tr>
+        <td>beat count</td>
+        <td>{{ data.lowlevel.rhythm.beats_count }}</td>
+      </tr>
+    </table>
+
+    {% if data.highlevel %}
+      <h3>High-level information</h3>
+
+      {% macro print_row(row) %}
+        {# Prints <name, value, probability> table row. #}
+          {% set probability = row[2]|float * 100 %}
+          <tr class="{{ 'success' if probability > 75 else (
+                        'warning' if probability > 25 else
```

Comment:
We definitely don't need this class on the entire row. Just because The Machine has a confidence of >75% of a value doesn't mean we have to say "LOOK, WE HAVE A SAD SONG!". I wonder if we could highlight just the probability number, but I'm tending towards not highlighting anything. In the end we should stop relying on this value and only show data which we're highly confident of.